src/dynodeController.ts

```diff
@@ -76,7 +82,9 @@ class NetworkController {
           return result;
         }),
       };
-      const nodeUI = new NodeUI(stage, nodeConfig);
+      const nodeTypeName = nodeModel.nodeType?.name || 'default';
```

Comment:
Many different node types at model side will share the same default UI class. But their names will be different: arithmetic, constant, etc. So, the model node type is not the same concept that the nodeUI type is. The node (model) type name mapping to nodeUI (view) type name should be maintained by a controller structure `{ modelType: viewType }`. A new NetworkController method will be needed to register these mappings.